README.md

```diff
@@ -8,7 +8,7 @@ Full documentation can be found [here](https://docs.threshold.network/app-develo
 >
 > `nucypher-ts` is under [active development](https://github.com/nucypher/nucypher-ts/pulls):
 >
-> - SDK does not support policy revocation.
+> - SDK does not support access revocation.
```

Comment:
I think we can safely remove this now?  It's not s true statement anymore since access can be revoked by way of policy expiration or unsatisfied on-chain condition.